File: iniciante/python/1181-linha-na-matriz.py
def linha_na_matriz():
    numero_da_linha = int(input())
    operacao = input()
    matriz = list()
    linha = list()
    for i in range(12):
        for j in range(12):
            linha.append(float(input()))
        matriz.append(linha)
        linha = list()
    if operacao == 'S':
        soma = sum(matriz[numero_da_linha])
        print(f'{soma:.1f}')
    elif operacao == 'M':
        media = sum(matriz[numero_da_linha])/len(matriz[numero_da_linha])
        print(f'{media:.1f}')


linha_na_matriz()

File: iniciante/c/1181-linha-na-matriz.c
#include <stdio.h>

int main(){
    double matriz[12][12], soma = 0.0, media; 
    int i, j, linha;
    char operacao;
    scanf("%d %c", &linha, &operacao);
    for(i = 0; i < 12; i++){
        for(j = 0; j < 12; j++){
            scanf("%lf", &matriz[i][j]);
        }
    }
    for(i = 0; i < 12; i++){
        soma += matriz[linha][i];
    }
    if (operacao == 'S'){
        printf("%.1lf\n", soma);
    }
    else{
        media = soma / 12.0;
        printf("%.1lf\n", media);
    }
    return 0;
}
